cmdlist.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"os"
+
+	"github.com/rogpeppe/modcop/depfile"
+)
+
+var listCommand = &Command{
+	Short:     "list dependencies",
+	UsageLine: "list [packages]",
+	Long: `
+List shows the set of current dependencies of the listed packages.
+By default, only build dependencies are listed; if the
+-test flag is provided, test dependencies will be listed too.
+If the -m flag is provided, dependencies will be printed at the
+module level and version information will be printed too,
+in the same format used by "go list -m".
+
+List accepts the same package wildcard patterns that are accepted
```

Comment:
You could copy what `go list` does in this respect:

```
For more about specifying packages, see 'go help packages'.
```

Reply:
done